kodi_addon_checker/check_repo.py

```diff
@@ -12,7 +12,9 @@ def check_repo(repo_path, branch_name, all_repo_addons, pr, config):
     for addon_folder in toplevel_folders:
         if addon_folder[0] != '.':
             addon_path = os.path.join(repo_path, addon_folder)
-            addon_report = check_addon.start(addon_path, branch_name, all_repo_addons, pr, config)
-            repo_report.add(addon_report)
-
+            try:
+                addon_report = check_addon.start(addon_path, branch_name, all_repo_addons, pr, config)
+                repo_report.add(addon_report)
+            except (IOError, SyntaxError, RuntimeError, TypeError):
+                repo_report.add(Record(PROBLEM, "There is some issue with your addon"))
```

Comment:
@Razzeee do we want to blame the addon?
can the backtrace be reported somewhere?

Reply:
If not the addon then maybe something general like `Something went wrong` or something along that line.

Reply:
yes, as long as it counted as error/problem
why not catch all exceptions?

Reply:
By all exception you mean bare except i.e `except exception as e` ?